scripts/combine/setupCombine.py

```diff
@@ -652,7 +683,7 @@ def main(args, xnorm=False):
         logger.warning("For now setting --theoryAgnostic activates --unfolding, they should do the same things")
         if args.genAxes is None:
             args.genAxes = ["ptVgenSig", "absYVgenSig", "helicitySig"]
-            logger.warning("Automatically setting '--genAxes ptVgenSig absYVgenSig helicity' for theory agnostic analysis")
+            logger.warning("Automatically setting '--genAxes ptVgenSig absYVgenSig helicitySig' for theory agnostic analysis")
```

Comment:
Expand the above variable rather than hardcoding to avoid them getting out of sync as they did before

Reply:
yes, thanks